app/helpers/container_image_helper/textual_summary.rb

```diff
@@ -79,5 +79,41 @@ def textual_openscap_failed_rules_high
       high = failed_rules_summary[:High]
       {:label => _("High"), :value => high} if high
     end
+
+    def textual_exposed_ports
+      {:label => _("Exposed Ports"),
+       :value => (@record['exposed_ports'].collect { |p,t| "#{p}/#{t}" }).join(', ')
+      }
+    end
+
+    def method_missing(mthd_sym, *args, &bolck)
+      match = /textual_(.*)/.match(mthd_sym)
+      if match && match.size > 1
+        prop = match[1]
+        value = @record[prop] if @record.attribute_present?(prop)
+        value = value.join(' ') if value.kind_of?(Array)
+        {:label => _(prop.titlecase), :value => value}
+      else
+        super
+      end
+    end
+  end
+
+  def textual_group_env
+    {
+      :additional_table_class => "table-fixed",
+      :labels                 => [_("Name"), _("Type"), _("Value")],
+      :values                 => collect_env
+    }
+  end
+
+  def collect_env
+    @record['environment_variables'].collect do |name, value|
+      [
+        name,
+        value.nil? ? "REFERENCE" : "VALUE",
+        {:value => value, :expandable => true}
+      ]
+    end
```

Comment:
I think this already exists for containers. so we can reuse the code from there by moving it to container_summary_helper.